options/locale/locale_en-US.ini

```diff
@@ -138,6 +138,8 @@ confirm_delete_selected = Confirm to delete all selected items?
 name = Name
 value = Value
 
+default = Default
```

Comment:
I do not think it is a general translation, especially when it is used in a label.

For example, GitHub show "default" but not "Default"

Reply:
Did some changes in https://github.com/go-gitea/gitea/pull/26697/commits/284d6449fcae8f1f2c9f54bedd657a9a850d99f9